src/displayapp/screens/WatchFaceBinary.cpp

```diff
@@ -0,0 +1,693 @@
+	lv_obj_set_size(monLED0, LED_SIZE2, LED_SIZE2);
+	lv_obj_align(monLED0, monLED1, LV_ALIGN_OUT_RIGHT_MID, LED_SPACE_H2, 0);	
+
+
+
+
+
+
+
+
+	
+}
+
+WatchFaceBinary::~WatchFaceBinary()
+{
+	lv_obj_clean(lv_scr_act());
+}
+
+bool WatchFaceBinary::Refresh()
+{
+	//lv_label_set_text(label_test, "TEST");
+	
+	batteryPercentRemaining = batteryController.PercentRemaining();
+	if (batteryPercentRemaining.IsUpdated()) 
+	{
+		auto batteryPercent = batteryPercentRemaining.Get();
+		lv_label_set_text(batteryIcon, BatteryIcon::GetBatteryIcon(batteryPercent));
+		auto isCharging = batteryController.IsCharging() || batteryController.IsPowerPresent();
+		lv_label_set_text(batteryPlug, BatteryIcon::GetPlugIcon(isCharging));
+	}
+
+	batteryVoltage = batteryController.Voltage();
+	if (batteryVoltage.IsUpdated())
+	{
+		auto batteryVolt = batteryVoltage.Get();
+		lv_label_set_text_fmt(label_volt, "%1i.%02i V", batteryVolt / 1000, (batteryVolt % 1000 / 10));
+	} 
+
+	bleState = bleController.IsConnected();
+	if (bleState.IsUpdated())
+	{
+		if (bleState.Get() == true)
+		{
+			lv_label_set_text(bleIcon, BleIcon::GetIcon(true));
+		}
+		else
+		{
+			lv_label_set_text(bleIcon, BleIcon::GetIcon(false));
+		}
+	}
+	lv_obj_align(batteryIcon, lv_scr_act(), LV_ALIGN_IN_TOP_RIGHT, -5, 2);
+	lv_obj_align(label_volt, batteryIcon, LV_ALIGN_OUT_LEFT_MID, -5, 0);
+	lv_obj_align(batteryPlug, label_volt, LV_ALIGN_OUT_LEFT_MID, -5, 0);
+	lv_obj_align(bleIcon, batteryPlug, LV_ALIGN_OUT_LEFT_MID, -5, 0);
+
+
+	notificationState = notificatioManager.AreNewNotificationsAvailable();
+	if (notificationState.IsUpdated())
+	{
+		if (notificationState.Get() == true)
+			lv_label_set_text(notificationIcon, NotificationIcon::GetIcon(true));
+		else
+			lv_label_set_text(notificationIcon, NotificationIcon::GetIcon(false));
+	}
+
+	currentDateTime = dateTimeController.CurrentDateTime();
+
+	if (currentDateTime.IsUpdated()) 
+	{
+		auto newDateTime = currentDateTime.Get();
+
+		auto dp = date::floor<date::days>(newDateTime);
+		auto time = date::make_time(newDateTime - dp);
+		auto yearMonthDay = date::year_month_day(dp);
+
+		auto year = (int) yearMonthDay.year();
+		auto month = static_cast<Pinetime::Controllers::DateTime::Months>((unsigned) yearMonthDay.month());
+		auto day = (unsigned) yearMonthDay.day();
+		auto dayOfWeek = static_cast<Pinetime::Controllers::DateTime::Days>(date::weekday(yearMonthDay).iso_encoding());
+
+		int hour = time.hours().count();
+		auto minute = time.minutes().count();
+		auto second = time.seconds().count();
+
+		char secondsChar[3];
+		sprintf(secondsChar, "%02d", static_cast<int>(second));
+
+		char minutesChar[3];
+		sprintf(minutesChar, "%02d", static_cast<int>(minute));
+
+		char hoursChar[3];
+		sprintf(hoursChar, "%02d", hour);
+
+
+		// Binary Watch //
+		//////////////////
+	
+		if (hoursChar[0] != displayedChar[0] || hoursChar[1] != displayedChar[1] || minutesChar[0] != displayedChar[2] || minutesChar[1] != displayedChar[3])
```

Comment:
Are these leftovers? Are there more leftovers?

Reply:
sorry, i thought, this was needed for time-update. changed code to use only minutes.